draft-ietf-quic-transport.md

```diff
@@ -1232,6 +1232,15 @@ client MUST terminate with a VERSION_NEGOTIATION_ERROR error code if
 version negotiation occurred but it would have selected a different version
 based on the value of the supported_versions list.
 
+When an endpoint accepts multiple QUIC versions, it can potentially interpret
+transport parameters as they are defined by any of the QUIC versions it
+supports. Since the QUIC protocol relies on transport parameters for
+authenticating the version, the position and the format of the version fields in
+transport parameters MUST be consistent across all QUIC versions supported by
+the client. If a version of QUIC needs to define a new format for transport
+parameters, it MUST define and use a new TLS extension for transport parameters
```

Comment:
This isn't defined as a TLS extension in this document, but the format is specified as the layout for in the TransportParameters field in the quic_transport_parameters TLS extension (se line 1076 above.) I think this text probably belongs there, along with the description of the wire layout, instead of here with version negotiation. I think the text is fine, but I would move it up there.